packages/extensions/core/src/app.ts

```diff
@@ -572,7 +573,7 @@ async function resourceSchemaBatch(api: AutoRest): Promise<number> {
       subscribeMessages(instance, () => exitcode++);
 
       // set configuration for that item
-      instance.AddConfiguration(ShallowCopy(batchContext, "input-file"));
+      instance.AddConfiguration(omit(batchContext, "input-file"));
```

Comment:
What's omit do in this case?

Reply:
it creates a copy of the object minus the requested params(in this case ("input-file")